SPECS/ruby/ruby.spec

```diff
@@ -408,6 +410,9 @@ sudo -u test make test TESTS="-v"
 %{_rpmconfigdir}/rubygems.con
 
 %changelog
+* Mon Nov 4 2024 Saul Paredes <saulparedes@microsoft.com> - 3.1.4-8
```

Comment:
to fit with the rest of the changelog, could you use 2 digits for the date? (`Nov 04`)

Reply:
Updated!